cfgov/cfgov/urls.py

```diff
@@ -427,9 +427,9 @@
 
     url(r'^_status/', include_if_app_enabled('watchman', 'watchman.urls')),
 
-    flagged_url('FWB_RELEASE',
-                r'^(?i)consumer-tools/financial-well-being/',
-                include('wellbeing.urls')
+    url(
+        r'^(?i)consumer-tools/financial-well-being/',
+        include_if_app_enabled('wellbeing.urls')
```

Comment:
Could we get a BEWD to comment on whether we should have done this in the first place, or if it matters at all? Jimmy and I weren't sure.

Reply:
Pinging @cfpb/cfgov-backends for a quick opinion on this.